pkg/operator/controller/ingress/deployment_test.go

```diff
@@ -381,6 +389,11 @@ func TestDesiredRouterDeployment(t *testing.T) {
 
 	checkDeploymentHasEnvVar(t, deployment, "ROUTER_IP_V4_V6_MODE", true, "v4v6")
 
+	// Any value for loadBalancingAlgorithm other than "leastconn" should be
+	// ignored.
+	ci.Spec.UnsupportedConfigOverrides = runtime.RawExtension{
+		Raw: []byte(`{"loadBalancingAlgorithm":"random"}`),
```

Comment:
nit: Might want to use a value other than `random` here (ie `foo` or something unrelated to `loadBalancingAlgorithm`).

Reply:
Fixed.  Thanks!